regparser/notice/changes.py

```diff
@@ -240,6 +241,31 @@ def remove_intro(l):
     return l.replace('[text]', '')
 
 
+def pretty_change(change):
+    """Pretty print output for a change"""
+    node = change.get('node')
+    field = change.get('field')
+    if change['action'] == Verb.PUT and field:
+        return '%s changed to: %s' % (field.strip('[]').title(),
+                                      node.get('title', node['text']))
+    elif change['action'] in (Verb.PUT, Verb.POST):
+        verb = 'Modified' if change['action'] == Verb.PUT else 'Added'
+        if node.get('title'):
+            return verb + ' (title: %s): %s' % (node['title'], node['text'])
+        else:
+            return verb + ": " + node['text']
+    elif change['action'] == Verb.DELETE:
+        return 'Deleted'
+    elif change['action'] == Verb.DESIGNATE:
+        return 'Moved to ' + '-'.join(change['destination'])
+    elif change['action'] == Verb.RESERVE:
+        return 'Reserved'
+    elif change['action'] == 'KEEP':
```

Comment:
KEEP doesn't seem to be a Verb (as in Verb.KEEP). Should it be? Even if it's a no-op?